### PR TITLE
Add Hyrule Warriors Legends

### PR DESCRIPTION
**Pull request by LCypher00**
Adds Hyrule Warriors Legends.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by LCypher00** [comment]
Very sorry for openning a new PR, i'm still figuring out how github works and deleted my fork before the old PR was merged .

File: games/hyrule-warriors-legends/game.dat [hunk 1]
```diff
@@ -0,0 +1,30 @@
+title = "Hyrule Warriors Legends"
+description = "Hyrule Warriors Legends is a Hack & Slash game with RPG elements set in the universe of The Legend of Zelda. It was developed by Omega Force the team behind Dynasty Warriors and Team Ninja responsible for Ninja gaiden."
+github_issues = [ 2198]
```
Comment (Schplee) [inline games/hyrule-warriors-legends/game.dat]: There's a space before `2198`

Comment (Leo626) [inline games/hyrule-warriors-legends/game.dat]: `[2198]` there's a space in the space in the beginning

**Comment by Leo626** [comment]
LGTM

**Comment by Schplee** [comment]
LGTM

**Pull request merged**